apps/site/lib/site_web/channels/user_socket.ex

```diff
@@ -2,7 +2,8 @@ defmodule SiteWeb.UserSocket do
   use Phoenix.Socket
 
   ## Channels
-  channel("vehicles:*", SiteWeb.VehicleChannel)
+  channel("vehicles:*", SiteWeb.VehicleMapMarkerChannel)
```

Comment:
I agree! I was wary of changing the existing channel name as part of this PR to avoid issues during a deploy since the React code currently expects these messages via the "vehicles:" channel. I suppose I could add an additional `vehicle-map-markers-` channel and add frontend handling for that first... though I'll try to think if there is a simpler approach!

Reply:
Ah, that's a good point, I wasn't thinking about that. Yeah, that could work, I'll let you decide if it's even worth it or not.

Reply:
With the ideal that the old "vehicles:*" channel might eventually be removed if we standardize on building map markers in the view, I'm going to opt to leave this as-is and make a tech debt ticket for eventually removing it altogether.